EG23.tex

```diff
@@ -180,30 +196,54 @@ \subsection{Reactors}
-and discharge rates \cite{B.Feng_calculation}.  For the SFR 1 facility type, the cycle
-length is specified to be 5.44 y (for discharge burnup of 37.62 GWd/t), 5.41 y
-has been used (5 $\times$ 13 months). For SFR 2, 4.5y has been used (3 $\times$ 18 months)
-where 4.44 was expected.  With Cyclus' 1-month time step, the burnup resolution
-can be easly calculated as:
-
-\begin{equation}
-  \Delta BU = \frac{P_{th} \times \Delta t}{M_{r}},
-\end{equation}
-
-\noindent where $P_{th}$ is the thermal power of the reactor, $M_{r}$ its mass
-in heavy metal, $\Delta BU$ the burnup achievable precision and $\Delta t$ the
-time step size.  In this simulation the $\Delta BU$ are for the SFR 1 \& 2
-respectively: $0.73$ and $0.79~GWd/t$.  This does not affect the composition
-of the discharge fuel as the reactor model used for this analysis does not
-perform a burnup evolution calculation.  Only the mass of discharged fuel is
-affected with a relative error of :
-
-\begin{equation}
-  \frac{\Delta M}{M} = \frac{\Delta BU}{BU}.
-\end{equation}
+Because of the 1-month timestep of Cyclus, the batch quantity and cycle length
+parameters have been manipulated to match as closely as possible important
+invariants such as fuel residence time, burnup, and discharge rates
+\cite{B.Feng_calculation}.
+
+For the SFR 1 facility type, the cycle length is specified to be $5.44~$y (for
+discharge burnup of 37.62 GWd/t), $5.40~$y has been used (27/5 $\times$ 12
+months). For the SFR 2, $4.4~$y has been used (22/5. $\times$ 12 months) where
+4.44 was expected. 
+
+Because the cycle length (in month), the number of assembly in a core and the number of
+assembly in a batch are defined as integer in Cyclus, have an exact match between
+the specification could need some tuning and can be unefficient. For exemple, in
+the precedent exercice, for the SFR 1, one could have set $544~$assembly in the
+core, $544$ in the batch with a cycle lengh of 12 month to have an exact match
+with the specification. Such implementation would have been very long to run,
+because the DRE (Dynamic Ressource Exchange) would have to deal at each refilling of each reactor 100 assembly
+trades, so few 10000 trades each years.
+
+The DRE is one of the main part of Cyclus, it deals the exchange of material between
+the different archetypes (Facilities, Institutions, Regions...). The material
+exhcange in Cyclus is devided in 3 phases: 
+\begin{itemize}
+  \item phase 1, request: each archetypes send there material request to the DRE
+    corresponding to their individuals needs,
+  \item phase 2, bid: the DRE share the request informations to all the
+    facilities, allowing them to place a bid on some depending on there indivudual productions,
+  \item phase 3, trade: the DRE solve the request/bid problem and each facilities
+    apply the results.
+\end{itemize}
+
+This DRE system allows Cyclus to deal with competition in the supply of materials. 
+
```

Comment:
I remove the formula since it was not correct,
I update the text to detail how to set the correct specifications,

I also add a small part on DRE